tests/components/hue/test_bridge.py

```diff
@@ -120,3 +122,131 @@ async def test_handle_unauthorized(hass):
     assert hue_bridge.authorized is False
     assert len(mock_create.mock_calls) == 1
     assert mock_create.mock_calls[0][1][1] == "1.2.3.4"
+
+
+GROUP_RESPONSE = {
+    "group_1": {
+        "name": "Group 1",
+        "lights": ["1", "2"],
+        "type": "LightGroup",
+        "action": {
+            "on": True,
+            "bri": 254,
+            "hue": 10000,
+            "sat": 254,
+            "effect": "none",
+            "xy": [0.5, 0.5],
+            "ct": 250,
+            "alert": "select",
+            "colormode": "ct",
+        },
+        "state": {"any_on": True, "all_on": False},
+    }
+}
+SCENE_RESPONSE = {
+    "scene_1": {
+        "name": "Cozy dinner",
+        "lights": ["1", "2"],
+        "owner": "ffffffffe0341b1b376a2389376a2389",
+        "recycle": True,
+        "locked": False,
+        "appdata": {"version": 1, "data": "myAppData"},
+        "picture": "",
+        "lastupdated": "2015-12-03T10:09:22",
+        "version": 2,
+    }
+}
+
+
+async def test_hue_activate_scene(hass, mock_api):
+    """Test successful hue_activate_scene."""
+    config_entry = config_entries.ConfigEntry(
+        1,
+        hue.DOMAIN,
+        "Mock Title",
+        {"host": "mock-host", "username": "mock-username"},
+        "test",
+        config_entries.CONN_CLASS_LOCAL_POLL,
+        system_options={},
+        options={CONF_ALLOW_HUE_GROUPS: True, CONF_ALLOW_UNREACHABLE: False},
+    )
+    hue_bridge = bridge.HueBridge(hass, config_entry)
+
+    mock_api.mock_group_responses.append(GROUP_RESPONSE)
+    mock_api.mock_scene_responses.append(SCENE_RESPONSE)
+
+    with patch("aiohue.Bridge", return_value=mock_api), patch.object(
+        hass.config_entries, "async_forward_entry_setup"
+    ):
+        assert await hue_bridge.async_setup() is True
+
+    assert hue_bridge.api is mock_api
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)